### PR TITLE
Add scPoli

### PR DESCRIPTION
**Pull request by moinfar**

### CODE REVIEW

File: scarches/trainers/scpoli/trainer.py [hunk 1]
```diff
@@ -0,0 +1,609 @@
...
+            self.landmarks_labeled_cov = self.model.landmarks_labeled["cov"]
+        if self.landmarks_labeled is not None:
+            self.landmarks_labeled = self.landmarks_labeled.to(device=self.device)
+            self.landmarks_labeled_cov = self.landmarks_labeled_cov.to(
+                device=self.device
+            )
+
+    def train(self, n_epochs=400, lr=1e-3, eps=0.01):
+        begin_loader = time.time()
+        print("loaders init")
```
Comment (cdedonno) [inline scarches/trainers/scpoli/trainer.py:127]: @moinfar There are some lines here that are leftovers from some debugging i was doing

Comment (cdedonno) [reply scarches/trainers/scpoli/trainer.py:127]: Can you delete line 127, 129, 130, 131?

**Review by cdedonno — COMMENTED** [review]
@moinfar There are some debugging line leftovers, could you delete them please?

File: scarches/trainers/scpoli/trainer.py (hunk 1 above)
Comment (cdedonno) [reply scarches/trainers/scpoli/trainer.py:127]: also 126 is not needed

Comment (moinfar) [reply scarches/trainers/scpoli/trainer.py]: Done.

**Comment by M0hammadL** [comment]
we need to add a proper tutorial here with text to guide the user in every step with explanations @cdedonno  @moinfar

**Comment by Koncopd** [comment]
Finished?

**Comment by cdedonno** [comment]
More or less, let's wait for biorxiv submission and then you can merge

**Comment by cdedonno** [comment]
@Koncopd @M0hammadL I think we can merge the PR now

**Pull request merged**